wal_e/blobstore/swift/utils.py

```diff
@@ -34,7 +34,8 @@ def uri_put_file(creds, uri, fp, content_type=None):
     conn = calling_format.connect(creds)
 
     conn.put_object(
-        container_name, url_tup.path, fp, content_type=content_type
```

Comment:
If you do lstrip, doesn't that make it impossible for a user with existing archives to continue to put them in the same spot by adding an extra slash to the config?

Reply:
@fdr fixed it (sorry for delay)